docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/shape/squeeze-15.rst

```diff
@@ -0,0 +1,104 @@
+Squeeze
+=======
+
+
+.. meta::
+  :description: Learn about Squeeze-15 - a shape manipulation operation, which
+                can be performed on one required and one optional input tensor.
+
+**Versioned name**: *Squeeze-15*
+
+**Category**: *Shape manipulation*
+
+**Short description**: *Squeeze* removes dimensions equal to 1 from the first input tensor.
+
+**Detailed description**: *Squeeze* can be used with or without the second input tensor.
+
+* If only the first input is provided, every dimension that is equal to 1 will be removed from it.
+* With the second input provided, each value is an index of a dimension from the first tensor that is to be removed. Specified dimension should be equal to 1, otherwise it will be ignored and copied as is.
+  Dimension indices can be specified directly, or by negative indices (counting dimensions from the end).
+
+.. note::
+
+    Behavior before 2024.3 OpenVINO release: Error is raised when dimension to squeeze is not compatible with 1.
```

Comment:
Is this mentioned version of OV correct? If I understand correctly 24.4 does not include updated v15::Squeeze op

Reply:
Done